src/main/java/seedu/address/model/date/Date.java

```diff
@@ -8,12 +8,12 @@
 import java.time.format.DateTimeParseException;
 
 /**
- * Represents a Person's date of birth in the address book.
+ * Represents a date in the application.
  * Guarantees: immutable; is valid as declared in {@link #isValidDate(String)}
  */
-public class DateOfBirth implements Comparable<DateOfBirth> {
+public class Date implements Comparable<Date> {
 
-    public static final String MESSAGE_CONSTRAINTS = "Date of birth must be in format: dd/mm/yyyy";
+    public static final String MESSAGE_CONSTRAINTS = "Dates must be in format: dd/MM/yyyy";
```

Comment:
Is there a reason why the month `MM` is uppercase but day and year are not?

Reply:
Ahh I think I didn't catch this. I think I will change it to all uppercase instead as I was confusing it with the DateTimeFormatter (where the cases matter).